spec/index.bs

```diff
@@ -2226,54 +2222,56 @@ interface GPUTextureUsage {
-                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=] must be less than or equal to
-                                        |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
-                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be less than or equal to
-                                        |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
-                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be less than
-                                        or equal to |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
-                                    :: |descriptor|.{{GPUTextureDescriptor/sampleCount}} must be 1.
-                                    :: |descriptor|.{{GPUTextureDescriptor/format}} must not be a compressed format or depth/stencil format.
+                                    ::
+                                        - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=] must be less than or equal to
+                                            |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                        - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be less than or equal to
+                                            |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                        - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be less than
+                                            or equal to |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                        - |descriptor|.{{GPUTextureDescriptor/sampleCount}} must be 1.
+                                        - |descriptor|.{{GPUTextureDescriptor/format}} must not be a compressed format or depth/stencil format.
                                 </dl>
-                            - If |descriptor|.{{GPUTextureDescriptor/sampleCount}} > 1 (|descriptor|.{{GPUTextureDescriptor/sampleCount}} is 4):
+                            - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=] must be multiple of [=texel block width=].
+                            - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be multiple of [=texel block height=].
+
+                            - If |descriptor|.{{GPUTextureDescriptor/sampleCount}} > 1:
                                 - |descriptor|.{{GPUTextureDescriptor/mipLevelCount}} must be 1.
                                 - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be 1.
                                 - |descriptor|.{{GPUTextureDescriptor/usage}} must not include the {{GPUTextureUsage/STORAGE}} bit.
                                 - |descriptor|.{{GPUTextureDescriptor/format}} must be a [=renderable format=].
```

Comment:
I noticed there is still an `Issue` in `[[#texture-format-caps]]`: "Add multisampling to the tables below."
This makes that issue obsolete, right? If so, please remove it.

Reply:
Yeah. That issue is not necessary now if it can pass texture validation rules here. Done.